main/src/main/java/org/mobilitydata/gtfsvalidator/notice/UnusedShapeNotice.java

```diff
@@ -0,0 +1,16 @@
+package org.mobilitydata.gtfsvalidator.notice;
+
+import com.google.common.collect.ImmutableMap;
+
+public class UnusedShapeNotice extends Notice {
```

Comment:
For consistency with the other notices, I suggest reming this one to `UnusedShape`.

Another solution would be to append the term "notice" to the other notices' class name:
- `InconsistentAgencyField`
- `StartAndEndDateOutOfOrder`
- `StopTimeWithArrivalBeforePreviousDepartureTime`
- `StopTimeWithDepartureBeforeArrivalTime`
- `StopTimeWithOnlyArrivalOrDepartureTime`
- `UnusedShape`